figures/use-case-self-host.puml

```diff
@@ -3,12 +3,14 @@ left to right direction
 actor "Self-Hoster" as hoster
 actor "User" as user
 rectangle OTT {
-  usecase "Deploy Self-Hosted Instance" as UC1
-  usecase "Utilize Docker" as UC1.1
+  usecase "Has Docker Installed" as UC1
+  usecase "Has Docker-Compose Installed" as UC1.1
```

Comment:
These are prerequisites. Add a `prereq` arg to `usecase` to put it in the table

Use this for reference: https://www.usability.gov/how-to-and-tools/methods/use-cases.html